### PR TITLE
new topic for xsoar.pan.dev on indicator fields

### PR DESCRIPTION
**Pull request by tschanfeld**
new topic on indicator fields https://github.com/demisto/content/pull/17794

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by altmannyarden** [comment]
@ytauman FYI

File: docs/indicators/indicator-fields.md [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+| Expiration | The expiration date of the object |
+| Source Time Stamp | When the object was created in the system |
+| Modified | When the object was last modified |
+
+| Custom Fields - Core | Description |
+| ----------- | ----------- |
+| Blocked | A Boolean switch to mark the object as blocked in the user environment |
+| Community Notes | Comments and freeform notes regarding the indicator |
+| Description | The description of the object |
+| STIX ID | The STIX ID for the object in the format of `account--<UUID>` |
```
[the remark this answers is not in the document]
Comment (tschanfeld) [reply docs/indicators/indicator-fields.md:165]: type should be in brackets?

**Comment by tschanfeld** [comment]
@altmannyarden What happened with this?

**Comment by tschanfeld** [comment]
@altmannyarden

**Comment by tschanfeld** [comment]
leave open

**Comment by tschanfeld** [comment]
@altmannyarden

**Comment by glicht** [comment]
What is the status here?

**Comment by Ni-Knight** [comment]
@glicht - waiting on PMs (Hitesh and Pramukh).

**Comment by Ni-Knight** [comment]
Still waiting for letter to be sent to customers.

**Comment by MosheEichler** [comment]
@Ni-Knight What is the status here?

**Comment by Ni-Knight** [comment]
@MosheEichler - still waiting

**Comment by altmannyarden** [comment]
@Ni-Knight lets see it any changes are needed here and merge the PR (from the description, fields, etc.)

File: docs/indicators/indicator-fields.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ni-Knight) [reply docs/indicators/indicator-fields.md]: no

**Comment by Ni-Knight** [comment]
@tschanfeld - changed it a bit due to changes to the project, can you go over the first paragraph to make sure Its ok (nothing else changed).

**Review by Ni-Knight — APPROVED** [review]

**Pull request merged**